src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -274,13 +274,13 @@
     <value>Attributes can define mandatory arguments that must be specified when you apply the attribute to a target. These are also known as positional arguments because they are supplied to attribute constructors as positional parameters. For every mandatory argument, the attribute should also provide a corresponding read-only property so that the value of the argument can be retrieved at execution time. Attributes can also define optional arguments, which are also known as named arguments. These arguments are supplied to attribute constructors by name and should have a corresponding read/write property.</value>
   </data>
   <data name="DefineAccessorsForAttributeArgumentsMessageDefault" xml:space="preserve">
```

Comment:
Seems like lot of message resource strings in this file have "Message" within the resource name instead of at the end. Not sure if it is possible to rename these to move Message to the end without making the name meaningless.

Reply:
> This was one of the cases I was explaining in the ticket. Currently when we have a rule where the various descriptors share the same title and description but differ by the message we use the following pattern: . It would be easier for the analyzer if we were able to use a different pattern, for example <Message_>.

Reply:
Ignore this comment for this PR.